commit 1d9721ff73427d8558f5efaa78a78772b7d9db1e
Author: John Johansen <john.johansen@canonical.com>
Date:   Wed Sep 27 02:10:17 2017 -0400

    UBUNTU: SAUCE: LSM stacking: fixup apparmor stacking enablement
    
    AppArmor doesn't need to register twice.
    
    Signed-off-by: John Johansen <john.johansen@canonical.com>
    Signed-off-by: Seth Forshee <seth.forshee@canonical.com>

diff --git a/security/apparmor/lsm.c b/security/apparmor/lsm.c
index 9ddd1c418343..a031ed6401cb 100644
--- a/security/apparmor/lsm.c
+++ b/security/apparmor/lsm.c
@@ -1565,9 +1565,7 @@ static int __init apparmor_init(void)
 		return 0;
 	}
 
-	if (!apparmor_enabled ||
-	    !security_module_enable("apparmor",
-				IS_ENABLED(CONFIG_SECURITY_APPARMOR_STACKED))) {
+	if (!apparmor_enabled) {
 		aa_info_message("AppArmor disabled by boot time parameter");
 		apparmor_enabled = 0;
 		return 0;
